chapter11/answers.md

```diff
@@ -1,44 +1,46 @@
 # Chapter 11 Questions and Answers
 
-#### 1. How would you write a function called `lucy()` that just returns all the `NPC` types matching the name 'Lucy Westenra'?
+#### 1. How would you write a function called `get_lucies()` that just returns all the `Person` types matching the name 'Lucy Westenra'?
```

Comment:
Ah, unfortunately the previous question 1 had a cardinality mismatch that would have necessitated an assert_single which made it feel a bit too complicated for a Question 1 and I thought I would go for a `set of` function instead.

https://github.com/edgedb/easy-edgedb/issues/60

(I don't remember seeing the cardinality error before, not sure if I just missed it or if the EdgeDB compiler back then wasn't smart enough to know)

We could easily come up with an easier function for a new Question 1 though, what do you think of that? Never hurts to have an extra one.

Reply:
And I did have fun with pluralizing Lucy this way!

Reply:
> Ah, unfortunately the previous question 1 had a cardinality mismatch that would have necessitated an assert_single which made it feel a bit too complicated for a Question 1 and I thought I would go for a `set of` function instead.

Ah, I see. Yeah, that makes sense.

> We could easily come up with an easier function for a new Question 1 though, what do you think of that? Never hurts to have an extra one.

That might be nice if you think it makes sense. This new number 1 feels slightly more complicated, but so would a reworked number 1 with `assert_single`. Maybe a new one would the the right move...

Not strictly necessary, but maybe it would ease readers in a bit more gently.